FlopperamUnrealMCP/Plugins/UnrealMCP/Source/UnrealMCP/Private/Commands/BlueprintGraph/NodeManager.cpp

```diff
@@ -0,0 +1,631 @@
+/**
+ * Filename: NodeManager.cpp
+ * Creation Date: 2025-01-06
+ * Author: Zoscran
+ * Description: Implementation of Blueprint node management
+ */
+
+#include "Commands/BlueprintGraph/NodeManager.h"
+#include "Commands/BlueprintGraph/Nodes/ControlFlowNodes.h"
+#include "Commands/BlueprintGraph/Nodes/DataNodes.h"
+#include "Commands/BlueprintGraph/Nodes/UtilityNodes.h"
+#include "Commands/BlueprintGraph/Nodes/CastingNodes.h"
+#include "Commands/BlueprintGraph/Nodes/AnimationNodes.h"
+#include "Commands/BlueprintGraph/Nodes/SpecializedNodes.h"
```

Comment:
Hey, were you able to get this to compile successfully? I think you left out the Nodes folder here